apps/namada-interface/tsconfig.json

```diff
@@ -1,22 +1,23 @@
 {
   "compilerOptions": {
-    "baseUrl": "./src",
-    "target": "es2015",
-    "lib": ["dom", "dom.iterable", "esnext"],
     "allowJs": true,
-    "skipLibCheck": true,
-    "esModuleInterop": true,
     "allowSyntheticDefaultImports": true,
-    "strict": true,
+    "baseUrl": "./src",
+    "esModuleInterop": true,
+    "esModuleInterop": true,
```

Comment:
Not added by your PR but the sorting shows up a duplicate `esModuleInterop` key we should probably remove at some point. I'm wondering too if it would be a good idea to have a shared tsconfig in the future that each package can extend, but not sure.

Reply:
Ha did not notice that :D Thanks! Yeah one config makes sense